flexmeasures_openweathermap/cli/commands.py

```diff
@@ -139,7 +145,10 @@ def collect_weather_data(location, store_in_db, num_cells, method, region):
         raise Exception(
             "[FLEXMEASURES-OWM] Setting OPENWEATHERMAP_API_KEY not available."
         )
-    locations = get_locations(location, num_cells, method)
+    if asset_id is not None:
+        locations = get_asset_id_location(asset_id)
+    else:
```

Comment:
This could run into the situation that `location` is empty, as well. We need this to become `elif` and under `else` print a warning and exit  (as we have no location).